src/Octoshift/Services/WarningsCountLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace OctoshiftCLI.Services;
+
+public class WarningsCountLogger
+{
+    private readonly OctoLogger _log;
+
+    public WarningsCountLogger(OctoLogger logger)
+    {
+        _log = logger;
+    }
+
+    public void LogWarningsCount(int warningsCount)
+    {
+        switch (warningsCount)
+        {
+            case 0:
+                _log.LogInformation("No warnings encountered during this migration");
+                break;
+            case 1:
+                _log.LogWarning("1 warning encountered during this migration");
```

Comment:
I wonder if we should point to the Migration Log explicitly here. Maybe `LogWarningsCount` could also be passed the log URL, and we could print that in the same log message?

Reply:
The CLI writes this line after writing to the console about the warnings:

```
[2023-05-16 14:26:25] [INFO] Migration log available at  or by running `gh bbs2gh download-logs --github-target-org import-testing --target-repo ***2gh-ccf221ea-64e7-432c-8bfc-03bc195251aa`
```

It looks like there was a regression in the log output at some point, because the log URL is supposed to be present here:

https://github.com/github/gh-gei/blob/f99944f9caab330a493da25af99f2ffe3e3cf3c1/src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs#L166-L166

If this :point_up: was working and the URL is being written to the output, will this satisfy what you're looking for?

Reply:
If that worked, it would most definitely meet my needs and expectations 🙏🏻